test/service_tests/user_config_test.dart

```diff
@@ -78,7 +79,81 @@ void main() async {
     setUpAll(() {
       registerServices();
     });
-    test('Test for User log out.', () async {
+
+    test('Test for getters & setters.', () {
+      final model = UserConfig();
+
+      // model.currentOrgInfoController
+      expect(model.currentOrgInfoController, isA<StreamController<OrgInfo>>());
+
+      // model.currentOrgName
+      expect(model.currentOrgName, isA<String>());
+
+      // model.currenOrg (setter)
+      model.currentOrg = OrgInfo(name: 'org');
+
+      // print(model.currentOrgInfoController);
+    });
+
+    test('Test for userLoggedIn method.', () async {
+      final model = UserConfig();
+      model.currentUser.id = 'fake_id';
+
+      userBox.put('user', User(id: 'fake', firstName: 'first'));
+
+      final Map<String, dynamic> data = {
+        'users': [
+          {
+            '_id': '1234567890',
+            'firstName': 'John',
+            'lastName': 'Doe',
+            'email': 'johndoe@example.com',
+            'image': 'https://example.com/profile.jpg',
+            'accessToken': 'exampleAccessToken',
+            'refreshToken': 'exampleRefreshToken',
+          }
+        ],
+      };
+
+      when(
```

Comment:
These simulation `when`(s) can be called in `setupAll` or where the mock is defined.

Reply:
Alright